example_clients/gui/gui.html

```diff
@@ -2,38 +2,46 @@
 <html lang="en">
 	<head>
 		{% raw pyindi_head %}
-		<title>GUI</title>
+		<title>{{ title }}</title>
 		<script>
 			/**
 			* Runs on load. Builds the default properties using a callback.
 			*/
 			const main = () => {
 				var devices = {% raw json_encode(devices) %};
-				initialize(devices);
+				initialize(devices, customGui=true);
 				return;
 			}
 
 			/**
-			* This function will be called whenever an INDI property with the device 
+			* This function will be called whenever an INDI property with the device
 			*	{{ devices }} is received. This is where users should modify the
 			*	code to append to specific objects.
 			*
-			*	If wanting to use a custom GUI, enable in the configuration and provide the correct syntax for selecting 
+			*	If wanting to use a custom GUI, enable in the configuration and provide the correct syntax for selecting
 			* elements.
 			* @param {Object} indi Contains all information about INDI property
 			*/
 			const handleProperty = (indi) => {
-        // If definition property, build it
-				if (indi.op === IndiOp.DEFINITION) {
-          var vector = updater.custom(indi);
-          // If returned vector is null, then bad selector in the <body> or not found
-          if (!vector) {
-            return;
-          }
-          // Handle vector for the first time here
-				}
-				// Handle vector every time it comes in
-				var vector = updater.vector(indi);
+        let htmlElement = handle(indi);
+
+        // Bad selector, meaning the custom GUI selector not found
+        if (!htmlElement) {
+          return;
+        }
```

Comment:
If you are OK with more aggressive use of the console.debug, this could provide some clarity. 
```suggestion
//Selector not found.
if(!htmlElement) {
  tag_str = `<div data-custom-vector="${indi.name}" data-custom-device="${indi.device}"/>`
  console.debug(`Selector not found. To add vector to page use ${tag_str}`)
  return;
}
```

Reply:
Everything looks good to me. I have some thoughts on Config that might be better worked out in a new PR.

I think we don't want developers accessing the Config object directly or at least changing it directly. We should have something that allows them to change the configuration. Could it simply be arguments to the initialize function as is done with customGui?

Reply:
I like the idea of passing the simple configuration as arguments when we initialize. Our configuration is small, so this shouldn't be an issue.

Reply:
I made the changes that you recommended. I also found a bug in getting BLOBs because of the helpful console log issued when the element is not found!